make SND=1 fails on pipewire 1.6 with: error[E0609]: no field `data` on type `spa_pod_builder`

Hey, release v1.18.0 doesn't build for me:
```
export ZSTD_SYS_USE_PKG_CONFIG=1
make BLK=1 NET=1 EFI=0 GPU=1 SND=1 INPUT=1 VERBOSE=1
...
Compiling krun-input v0.1.0 (/build/libkrun/src/libkrun-1.18.0/src/input)
   Compiling krun-display v0.1.0 (/build/libkrun/src/libkrun-1.18.0/src/display)
error[E0609]: no field `data` on type `spa_pod_builder`
  --> /build/.cargo/registry/src/index.crates.io-1949cf8c6b5b557f/libspa-0.8.0/src/pod/builder.rs:38:25
   |
38 |         (*this).builder.data = (*this).data.as_mut_ptr().cast::<c_void>();
   |                         ^^^^ unknown field
   |
   = note: available field is: `_address`

error[E0609]: no field `size` on type `spa_pod_builder`
  --> /build/.cargo/registry/src/index.crates.io-1949cf8c6b5b557f/libspa-0.8.0/src/pod/builder.rs:39:25
   |
39 |         (*this).builder.size = (*this)
   |                         ^^^^ unknown field
   |
   = note: available field is: `_address`

For more information about this error, try `rustc --explain E0609`.
error: could not compile `libspa` (lib) due to 2 previous errors
warning: build failed, waiting for other jobs to finish...
make: *** [Makefile:207: target/release/libkrun.so.1.18.0] Error 101
```

--- a/src/devices/Cargo.toml
+++ b/src/devices/Cargo.toml
@@ -29,7 +29,7 @@ libc = ">=0.2.39"
 libloading = "0.8"
 log = "0.4.0"
 nix = { version = "0.30.1", features = ["ioctl", "net", "poll", "socket", "fs"] }
-pw = { package = "pipewire", version = "0.8.0", optional = true }
+pw = { package = "pipewire", version = "0.9.2", optional = true }
 rand = "0.9.2"
 thiserror = { version = "2.0", optional = true }
 virtio-bindings = "0.2.0"

--- a/src/devices/src/virtio/snd/audio_backends/pipewire.rs
+++ b/src/devices/src/virtio/snd/audio_backends/pipewire.rs
@@ -11,8 +11,8 @@ use std::{
 
 use log::debug;
 use pw::{
-    context::Context, core::Core, properties::properties, spa, sys::PW_ID_CORE,
-    thread_loop::ThreadLoop,
+    context::ContextRc, core::CoreRc, properties::properties, spa, sys::PW_ID_CORE,
+    thread_loop::ThreadLoopRc,
 };
 use spa::{
     param::{
@@ -74,27 +74,25 @@ unsafe impl Sync for PwBackend {}
 #[allow(clippy::non_send_fields_in_send_ty)]
 pub struct PwBackend {
     pub stream_params: Arc<RwLock<Vec<Stream>>>,
-    thread_loop: ThreadLoop,
-    pub core: Core,
+    thread_loop: ThreadLoopRc,
+    pub core: CoreRc,
     #[allow(dead_code)]
-    context: Context,
-    pub stream_hash: RwLock<HashMap<u32, pw::stream::Stream>>,
+    context: ContextRc,
+    pub stream_hash: RwLock<HashMap<u32, pw::stream::StreamRc>>,
     pub stream_listener: RwLock<HashMap<u32, pw::stream::StreamListener<i32>>>,
 }
 
 impl PwBackend {
     pub fn new(stream_params: Arc<RwLock<Vec<Stream>>>) -> Self {
-        pw::init();
-
         // SAFETY: safe as the thread loop cannot access objects associated
         // with the loop while the lock is held
-        let thread_loop = unsafe { ThreadLoop::new(Some("Pipewire thread loop"), None).unwrap() };
+        let thread_loop = unsafe { ThreadLoopRc::new(Some("Pipewire thread loop"), None).unwrap() };
 
         let lock_guard = thread_loop.lock();
 
-        let context = Context::new(&thread_loop).expect("failed to create context");
+        let context = ContextRc::new(&thread_loop, None).expect("failed to create context");
         thread_loop.start();
-        let core = context.connect(None).expect("Failed to connect to core");
+        let core = context.connect_rc(None).expect("Failed to connect to core");
 
         // Create new reference for the variable so that it can be moved into the
         // closure.
@@ -341,7 +339,7 @@ impl AudioBackend for PwBackend {
                 *pw::keys::MEDIA_CATEGORY => media_category,
             };
 
-            let stream = pw::stream::Stream::new(&self.core, stream_name, props)
+            let stream = pw::stream::StreamRc::new(self.core.clone(), stream_name, props)
                 .expect("could not create new stream");
 
             let streams = self.stream_params.clone();

